tokenserver/assignment/sqlnode/sql.py

```diff
@@ -655,15 +691,17 @@ def get_best_node(self, service):
         nodeid = row.id
         node = str(row.node)
 
-        # Update the node to reflect the new assignment.
-        # This is a little racy with concurrent assignments, but no big deal.
-        where = [nodes.c.service == service, nodes.c.node == node]
-        where = and_(*where)
-        fields = {'available': nodes.c.available - 1,
-                  'current_load': nodes.c.current_load + 1}
-        query = update(nodes, where, fields)
-        con = self._safe_execute(query, close=True)
-        con.close()
+        if not send_to_spanner:
+            # Update the node to reflect the new assignment.
+            # This is a little racy with concurrent assignments, but no big
+            # deal.
+            where = [nodes.c.service == service, nodes.c.node == node]
+            where = and_(*where)
+            fields = {'available': self._sqlfunc_max(nodes.c.available - 1, 0),
+                      'current_load': nodes.c.current_load + 1}
```

Comment:
I can see a case for not updating `available` on the spanner node, but tracking `current_load` might still be useful to us in the future. But I don't feel strongly either way.

Reply:
Yeah, I can kinda see that. adding it back in.